frameworks/digital-outcomes-and-specialists-2/manifests/edit_service.yml

```diff
@@ -0,0 +1,125 @@
+-
+  name: Lab name
+  editable: False
+  questions:
+    - serviceName
+
+- name: Service essentials
+  editable: False
+  questions:
+    - helpGovernmentImproveServices
+    - bespokeSystemInformation
+    - dataProtocols
+    - openStandardsPrinciples
+
+- name: User research participants essentials
+  editable: False
+  questions:
+    - anonymousRecruitment
+    - manageIncentives
+
+- name: Team capabilities
+  editable: False
+  edit_questions: True
```

Comment:
`edit_questions` is used by during the submission process to show links per-question rather than per-section (i.e. the one-question-per-page flow).